### PR TITLE
Do not need sestatus

### PR DESCRIPTION
**Pull request by lgranie**
Simple workaround if sestatus is not available

### CODE REVIEW

File: src/main/java/nl/lexemmens/podman/service/SecurityContextService.java [hunk 1]
```diff
@@ -67,13 +67,18 @@ private boolean isSELinuxEnabled() throws MojoExecutionException {
...
+            Optional<String> seLinuxStatus = seStatusCommand.execute()
+                    .stream()
+                    .filter(line -> line.contains("SELinux status"))
+                    .map(this::extractSeLinuxStatus)
+                    .findFirst();
 
-        return seLinuxStatus.map(seLinuxStatusString -> seLinuxStatusString.equals("enabled")).orElse(false);
+            return seLinuxStatus.map(seLinuxStatusString -> seLinuxStatusString.equals("enabled")).orElse(false);
+        } catch (MojoExecutionException mee) {
+            log.debug(mee.getMessage());
```
Comment (lexemmens) [inline src/main/java/nl/lexemmens/podman/service/SecurityContextService.java:79]: LGTM! Propose to add a custom debug message like: "SELinux is not installed or not available".
```suggestion
            log.debug("SELinux is not installed or not available", mee.getMessage());
```

**Comment by lexemmens** [comment]
My apologies for the delayed response. I have been super busy in the past few weeks. The changes look good to me. I did add a minor comment regarding the logging when SELinux is not availables.

**Comment by rajha-korithrien** [comment]
Are there any thoughts about moving this PR forward. Turns out it is exactly what is needed to make this useful plugin work on macOS again. Thanks!

**Comment by lexemmens** [comment]
> Are there any thoughts about moving this PR forward. Turns out it is exactly what is needed to make this useful plugin work on macOS again. Thanks!

Please hang in there for the weekend. I aim to resolve this PR and make a release this weekend :)

**Comment by lexemmens** [comment]
> Are there any thoughts about moving this PR forward. Turns out it is exactly what is needed to make this useful plugin work on macOS again. Thanks!

I released version 1.17.0 just a minute ago. Should be in Maven Central within 30 minutes :)

**Pull request merged**